src/main/kotlin/no/nav/klage/kaka/api/KabalController.kt

```diff
@@ -51,13 +51,15 @@ class KabalKvalitetsvurderingController(
         @PathVariable("id") kvalitetsvurderingId: UUID,
         @RequestParam temaId: String?,
         @RequestParam ytelseId: String?,
+        @RequestParam typeId: String?
     ): ValidationErrors {
         val innloggetSaksbehandler = tokenUtil.getIdent()
         val kvalitetsvurdering =
             kvalitetsvurderingService.getKvalitetsvurdering(kvalitetsvurderingId, innloggetSaksbehandler)
         val ytelseToUse = if (ytelseId != null) Ytelse.of(ytelseId) else Ytelse.OMS_OMP
-        //FIXME use type from input
-        return ValidationErrors(kvalitetsvurdering.getInvalidProperties(ytelseToUse, Type.KLAGE).map {
+        val typeToUse = if (typeId != null) Type.of(typeId) else Type.KLAGE
```

Comment:
Jeg ville vurdert `typeId?.let{ Type.of(it) } ?: Type.KLAGE` som kanskje er mer "idomatisk Kotlin", men if-else gjør også susen.